### ISSUE TITLE
[BUG]: Cosmos-Server Crashes and does not recover if URLs has an error

### ISSUE THREAD

**Issue by CropWatchDevelopment** [azukaar/Cosmos-Server#93]
### What happened?

Enter a bad proxy type in the URLs section and Cosmos will crash and become un-recoverable

### What should have happened?

It should not crash, or require a full delete and restart.
Perhaps check for bad values being entered before saving?

### How to reproduce the bug?

1. Go to URLs
2. Click on Create
3. Enter some sort of proxy values, I have been unable to figure out what is bad as cosmos crashes and I can't get it to load
4. really quickly delete /var/lib/cosmos and resetup before people complain (had to do this 2 times :( )

### System details

- OS: Ubuntu
- Version 0.9.21

**Comment by azukaar** [comment]
What do you mean by wrong "proxy type"?
Which value exactly requires validation?

![image](https://github.com/azukaar/Cosmos-Server/assets/7872597/86f2130f-8746-4aa9-8de1-7ef36858391b)

**Comment by CropWatchDevelopment** [comment]
Yes, I suspect that I accidentily put the server IP address there, and not the URL...
OR what I did was enter a url, but it wasn't setup in my dns server yet

**Comment by azukaar** [comment]
It shouldnt crash though, it should just have a broken URL... can you give me an example of something that crashes?

**Comment by CropWatchDevelopment** [comment]
If I get a chance I will try to re-create this. I need to first setup a less used server :)

**Comment by SpoopyBE** [comment]
When I was playing around with URLs, I got into a situation where the cosmos-server container would not start.
Regardless of the configuration, I would expect the routing to break, but the container should always be able to start.

What I did:
1. ServApps -> cosmos-server -> URLs -> New
2. enable "Use Host" using cosmos.myrealdomain.name

The docker logs showed:
`2023/12/25 18:37:30 [ERROR] Container tries to connect to a non existing Cosmos network, replacing it. : Error response from daemon: network auto not found
`
